src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -103,7 +104,8 @@ public RobotContainer() {
       driveTrain.setDefaultCommand(
           new DriveCommand(driveTrain, rightJoystick::getY, rightJoystick::getX));
     } else {
-      driveTrain.setDefaultCommand(driveTrain.stopCommand());
+      // driveTrain.setDefaultCommand(driveTrain.stopCommand());
```

Comment:
This code should not be push to main if this is being done. Instead of making a pull request, either convert this to a draft or close the PR and just use the branch.